packages/api-client/tests/test-granules.js

```diff
@@ -6,8 +6,8 @@ const granulesApi = require('../granules');
 
 test.before((t) => {
   t.context.testPrefix = 'unitTestStack';
-  t.context.granuleId = 'granule-1';
-  t.context.collectionId = 'fakeName___fakeVersion';
+  t.context.granuleId = randomId('gran/a-b-c-123');
```

Comment:
Without using the encoder method, did this cause the tests to fail in the same way PODAAC saw?

Reply:
No, because it's all being mocked out with fake callbacks. I briefly talked to @Jkovarik about this and we discussed whether or not we should introduce an integration test. We leaned toward not adding one but curious what your thoughts are.

Reply:
Hmm interesting. Is there an existing integration test that could be updated with a granule/collectionId containing slashes? I tend to agree that adding one for this purpose isn't fun for anyone or useful.

At the end of the day our only real concern here is saying that we were able to replicate the problem and we believe this fix... fixes. If modifying an existing int test isn't feasible we could also do a manual one-off test with a deployed stack.